microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -392,6 +390,19 @@ def add_sample_apps_to_build_list(self, apps: Union[List[str], None]) -> None:
         else:
             self._sample_apps_to_build = []
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super().__init__(*args, **kwargs)
+        self._dpdk_source = kwargs.pop("dpdk_source")
```

Comment:
Add a default value, so in some case, if the field is missing, the default value will be used.

Reply:
update